### PR TITLE
fix double posting avatars.

### PR DESCRIPTION
**Pull request by NeonLightPWO**
im bad i know

### CODE REVIEW

**Comment by TheFenderStory** [comment]
Whats the point of deleting the entry in config? It's going to be changed when the new avatar is set anyway so this code seems extraneous.

**Comment by ShadesOfBlack** [comment]
it double posts in the avatar folder apparently

**Comment by TheFenderStory** [comment]
I'll take a look at this tonight and see if i can recreate the problem.

**Comment by TheFenderStory** [comment]
I'm having difficulty recreating the issue. Any more details you can give me about it?

**Comment by ShadesOfBlack** [comment]
hence this pull request as a possible fix?

File: chat-plugins/customavatar.js [hunk 1]
```diff
@@ -46,10 +46,10 @@ exports.commands = {
 			const parts = target.split(',');
 			const userid = toId(target);
```
Comment (TheFenderStory) [inline chat-plugins/customavatar.js]: The target is split on line 46 so this will not work as intended.

**Comment by ShadesOfBlack** [comment]
,

**Comment by Lord-Haji** [comment]
@TheFenderStory I think we should make a check to block double setting of avatars not delete it and reset it ?

**Comment by TheFenderStory** [comment]
Why? There really isn't an issue doing it this way. If someone tries to set an avatar for someone who already has one, it will just delete their old one. I think that streamlines things a bit

**Pull request merged**